src/flake8_to_ruff/tool_configs.rs

```diff
@@ -0,0 +1,17 @@
+use std::path::Path;
+
+use anyhow::Result;
+
+use super::{black::Black, isort::Isort, parse_black_options, parse_isort_options};
+
+pub struct ToolConfigs {
+    pub black: Option<Black>,
+    pub isort: Option<Isort>,
+}
+
+pub fn parse_tool_configs<P: AsRef<Path>>(path: P) -> Result<ToolConfigs> {
+    Ok(ToolConfigs {
+        black: parse_black_options(path.as_ref())?,
+        isort: parse_isort_options(path)?,
```

Comment:
Should these both take `path.as_ref()`? (Does using `path` for both, without `.as_ref()`, lead to a borrowing error?)

Reply:
Potentially yes, I'm guessing if more external configurations were added here, we'd need to `.as_ref()` all except the last.

Without `.as_ref()` I get the following in `parse_external_config`:

<img width="833" alt="image" src="https://user-images.githubusercontent.com/803013/213901866-abbe98ef-7d43-450f-96ca-0c38b75e6e3e.png">